code/modules/mod/modules/modules_medical.dm

```diff
@@ -8,12 +8,12 @@
 /obj/item/mod/module/health_analyzer
 	name = "MOD health analyzer module"
 	desc = "A module installed into the glove of the suit. This is a high-tech biological scanning suite, \
-		allowing the user indepth information on the vitals and injuries of others even at a distance, \
+		allowing the user indepth information on the vitals and injuries of others even at a distance, including even chemical scans, \
```

Comment:
even twice is weird wording

Reply:
Apparently chem scans are also on normie analyzers so the change is going away